routes/api/stories.js

```diff
@@ -0,0 +1,69 @@
+//2
+const express = require('express');
+const router = express.Router();
+const createError = require('http-errors');
+const {asyncHandler} = require('../utils');
+const { PostLike } = require("../../db/models");
+const { restoreUser } = require('../../auth');
+
+/* Like a story */
+router.post('/stories/:id/likes', restoreUser, asyncHandler(async(req, res, next) => {
+
+        if (!res.locals.authenticated) {
+
+           /* If the user is not authenticated,
+              return a 401 error */ 
+           return next(createError(401));
+
+        } else if (res.locals.authenticated ) {
+            
+            /* If the user is authenticated,
+               create a new like */
+            const likedPost = await PostLike.create({
+                userId: res.locals.user.id,
+                postId: req.params.id
+            })
+        }
+
+        /* User has been authenticated, and
+           a like has been created.
+           Set 201 status code for response, and
+           send likedPost as json response. */
+        res.status(201).json(likedPost);
+
+    }));
+
+/* Unlike a story */
+router.delete('/stories/likes/:id', restoreUser, asyncHandler(async(req, res, next) => {
+        
+        /* Initialize like object */
+        const like = await PostLike.findByPk(req.params.id);
+
+        /* Set status 404; 
+           there is no like object */
+        if (!like) {res.status(404)};
```

Comment:
`res.status` just sets the header of the response. Sending it using `res.statusCode(404)` is an option, but I think it's best we utilize error handlers here. We'll also need to make sure that the rest of the code does not attempt to execute, so be sure to return in the if statement.

To utilize error handlers, use next with an error similar to when `res.locals.authenticated` is `false`.

Reply:
Code has been changed to use next and use error handler.